<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NBioBSP_DataConvert</class>
 <widget class="QDialog" name="NBioBSP_DataConvert">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>809</width>
    <height>442</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>809</width>
    <height>442</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>809</width>
    <height>442</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>NBioBSP_DataConvert</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>581</width>
     <height>71</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Test Operation</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rdExport">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>147</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Export Function Test</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rdImport">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>40</y>
      <width>147</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Import Function Test</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>85</y>
     <width>581</width>
     <height>121</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>71</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Data Type</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboDataType">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>561</width>
      <height>27</height>
     </rect>
    </property>
    <property name="maxVisibleItems">
     <number>20</number>
    </property>
    <item>
     <property name="text">
      <string>FDP(FDU)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>FDA</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>FDA(Old Version)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>FDAC</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>FIM10-HV / FIM10-LV</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>FIM01-HV</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>FIM01-HD</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>FeliCa</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Extension(Max 1024bytes)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Variable Size</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ANSI</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ISO</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>91</y>
      <width>171</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Template Size(Variable Size)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="editTemplateSize">
    <property name="geometry">
     <rect>
      <x>185</x>
      <y>85</y>
      <width>381</width>
      <height>27</height>
     </rect>
    </property>
    <property name="inputMask">
     <string>999; </string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>581</width>
     <height>101</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>121</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Export Function Test</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btExport">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Capture &amp; Export</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btSave">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>40</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>315</y>
     <width>581</width>
     <height>101</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>121</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Import Function Test</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btLoad">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>43</x>
      <y>40</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Load</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btImport">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>40</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Import &amp; Verify</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="labelStatus">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>420</y>
     <width>801</width>
     <height>20</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="text">
    <string>Status</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelFP">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>120</y>
     <width>200</width>
     <height>200</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::WinPanel</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
